docs/docs/clients/server-side.md

```diff
@@ -562,6 +554,18 @@ secret_button_feature_value = Flagsmith.Client.get_feature_value(flags, "secret_
 </TabItem>
 </Tabs>
 
+### When running in [Remote Evaluation mode](/clients/overview#remote-evaluation)
+
+- All Traits defined in the SDK will automatically be persisted against the Identity within the Flagsmith API.
```

Comment:
Tiny nit: This bullet point doesn't really clarify *during what use* Traits will be automatically persisted.